Intersect.Server/Entities/Npc.cs

```diff
@@ -1501,7 +1501,11 @@ public override void ProcessRegen()
             byte newDir,
             bool adminWarp = false,
             byte zOverride = 0,
-            bool mapSave = false
+            bool mapSave = false,
+            bool fromWarpEvent = false,
+            MapInstanceType mapInstanceType = MapInstanceType.NoChange,
```

Comment:
When NoChange is made to be 0, this can be changed to = default

Reply:
Done